packages/components/Menu/SPEC.md

```diff
@@ -218,6 +218,11 @@ export type MenuListProps = {
    */
   defaultChecked?: string[];
 
+  /**
+   * States that menu items can contain icons and reserves space for item alignment
+   */
```

Comment:
Should we also add a screenshot with icons so it's more obvious?

Reply:
Ideally yes, but I won't have time to update this for a while so I'd prefer to check this in as is.